src/expressions/match-expr.md

```diff
@@ -102,6 +111,21 @@ let message = match maybe_digit {
 };
 ```
 
+`if let` guards are introduced by the `if let` keywords and are composed of a pattern, an `=` and a scrutinee expression.
```

Comment:
```suggestion
`if let` guards are composed following [The Rust Reference `if let` keywords syntax](https://doc.rust-lang.org/stable/reference/expressions/if-expr.html#if-let-expressions).
```

Reply:
@kw-fn, I think the original is correct here, as it is just describing the syntax, and is a style used throughout the reference as a way to anchor the description of what the text is talking about.  Also, we do not use external links, or refer to the reference by name within itself.